### DISCUSSION TITLE
Twin macro doesn't work with custom colour from tailwind

### DISCUSSION THREAD

**Discussion by luuhn** [ben-rogerson/twin.macro#716] [General]
I tried to add new custom colour in tailwind config in extends as following
  extend: {
      colors: {
        primary: '#db00ff',
        ribbon: '#0047ff',
      },
But I got the error when using 
export const buttonVariantSuccess = tw`border-green-500 bg-primary text-white hover:bg-green-600`;

![image](https://user-images.githubusercontent.com/8158368/175326501-2266f26a-674b-40ab-8c10-ca422c78cc20.png)

This primary colour works well on className. I tried using theme, css with twin macro but it still the same. Only default colours from tailwind and direct colour hex code worked.
I used   "twin.macro": "^2.8.2", "tailwindcss": "^2.2"
Could you please help me on this issue?
Thanks,
Michael

**Comment by khzouroussama** [comment]
maybe you need to restart the dev server?

**Reply by Sowed** [reply]
@luuhn did you have this resolved?  I am using `v3.1.0` but still stuck with this issue. Restarting the server does not help @khzouroussama .

**Comment by TigranAvagyan18** [comment]
Having this issue too, any workaround?

**Reply by ben-rogerson** [reply]
Custom colors shouldn't have issues - couple questions:

- What does your twin config look like?
- And your tailwind.config?
- Which color are you trying to grab, and how?

**Comment by ben-rogerson** [comment]
Closing for now, feel free to reopen with additional info, eg: repo for testing this bug within.
